examples/fedora-coreos.yaml

```diff
@@ -0,0 +1,15 @@
+# This example requires Lima v0.7.0 or later.
+# <https://fedoraproject.org/coreos/download/>
+
+images:
+- location: "https://builds.coreos.fedoraproject.org/prod/streams/stable/builds/38.20230609.3.0/x86_64/fedora-coreos-38.20230609.3.0-qemu.x86_64.qcow2.xz"
+  arch: "x86_64"
+  digest: "sha256:2c039e778505f608ff78b8ba02d497780ee95182f477dbd939ddd335ece29326"
+- location: "https://builds.coreos.fedoraproject.org/prod/streams/stable/builds/38.20230609.3.0/aarch64/fedora-coreos-38.20230609.3.0-qemu.aarch64.qcow2.xz"
+  arch: "aarch64"
+  digest: "sha256:0410362c212df040096933738607e56027005deb1c4a3eae417079180aaadc65"
+
+# The guest home directory can not be changed with CoreOS currently.
```

Comment:
The host home directory can be still mounted?

Reply:
My problem was they both use `/home/anders`... baby there is a "homeDir" field in ignition - just that it looks unimplemented at the moment, and gives errors.

Another issue was that the "core" user now claims UID 1000, previously it used 500